src/webapp/CompositionRoot.tsx

```diff
@@ -8,38 +8,38 @@ import { CompleteUserProgressUseCase } from "../domain/usecases/CompleteUserProg
 import { GetModuleUseCase } from "../domain/usecases/GetModuleUseCase";
-import { GetSettingsPermissionsUseCase } from "../domain/usecases/GetSettingsPermissionsUseCase";
-import { GetShowAllModulesUseCase } from "../domain/usecases/GetShowAllModulesUseCase";
-import { ImportLandingPagesTranslationsUseCase } from "../domain/usecases/ImportLandingPagesTranslationsUseCase";
 import { SwapLandingChildOrderUseCase } from "../domain/usecases/SwapLandingChildOrderUseCase";
 import { ImportLandingPagesUseCase } from "../domain/usecases/ImportLandingPagesUseCase";
 import { ImportModulesUseCase } from "../domain/usecases/ImportModulesUseCase";
-import { ImportModuleTranslationsUseCase } from "../domain/usecases/ImportModuleTranslationsUseCase";
 import { InstallAppUseCase } from "../domain/usecases/InstallAppUseCase";
 import { ListDanglingDocumentsUseCase } from "../domain/usecases/ListDanglingDocumentsUseCase";
 import { ListInstalledAppsUseCase } from "../domain/usecases/ListInstalledAppsUseCase";
 import { ListLandingChildrenUseCase } from "../domain/usecases/ListLandingChildrenUseCase";
 import { ListModulesUseCase } from "../domain/usecases/ListModulesUseCase";
 import { ResetModuleDefaultValueUseCase } from "../domain/usecases/ResetModuleDefaultValueUseCase";
 import { SearchUsersUseCase } from "../domain/usecases/SearchUsersUseCase";
-import { SetShowAllModulesUseCase } from "../domain/usecases/SetShowAllModulesUseCase";
 import { SwapModuleOrderUseCase } from "../domain/usecases/SwapModuleOrderUseCase";
 import { UpdateLandingChildUseCase } from "../domain/usecases/UpdateLandingChildUseCase";
 import { UpdateModuleUseCase } from "../domain/usecases/UpdateModuleUseCase";
-import { UpdateSettingsPermissionsUseCase } from "../domain/usecases/UpdateSettingsPermissionsUseCase";
 import { UpdateUserProgressUseCase } from "../domain/usecases/UpdateUserProgressUseCase";
 import { UploadFileUseCase } from "../domain/usecases/UploadFileUseCase";
+import { GetConfigUseCase } from "../domain/usecases/GetConfigUseCase";
+import { Dhis2DocumentRepository } from "../data/repositories/Dhis2DocumentRepository";
+import { Instance } from "../data/entities/Instance";
+import { SaveConfigUseCase } from "../domain/usecases/SaveConfigUseCase";
+import { ExtractTranslationsUseCase } from "../domain/usecases/ExtractTranslationsUseCase";
+import { ImportTranslationsUseCase } from "../domain/usecases/ImportTranslationsUseCase";
 
 export function getCompositionRoot(baseUrl: string) {
+    const instance = new Instance({ url: baseUrl });
     const configRepository = new Dhis2ConfigRepository(baseUrl);
     const instanceRepository = new InstanceDhisRepository(configRepository);
     const trainingModuleRepository = new TrainingModuleDefaultRepository(configRepository, instanceRepository);
     const landingPageRepository = new LandingPageDefaultRepository(configRepository, instanceRepository);
```

Comment:
[Don't change] Just to mention, don't take this as an example because calling repositories from other repositories is indeed a code smell. We won't change this because we would spend too much time. With this PR and #212 we will be at least one step closer